### DISCUSSION TITLE
助教，我电脑里之前有anaconda，然后我按照b站的换源方法操作结果没有成功，今天我下载了miniconda，在下载的时候报错了，您看看怎么回事？

### DISCUSSION THREAD

**Discussion by nwssjzw** [datawhalechina/learn-python-the-smart-way-v2#25] [提问答疑]
![屏幕截图 2023-10-17 164723](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/147988477/2ccd3e98-4364-4bd8-8dee-0a6f6bc7e877)

**Comment by anine09** [comment]
首先你的提问不符合[提问规则](https://datawhalechina.github.io/learn-python-the-smart-way-v2/Question/question/)，下次请注意。

其次，我建议你把 Anaconda 卸载掉，只装 Miniconda，我们可能不想花时间为你解决冲突问题。

如果还是出现错误，请将你的 log 提交到[公共邮箱](mailto:learn.python.the.smart.way@gmail.com)
